e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -363,4 +363,20 @@ describeWithSnowplow("scenarios > dashboard > tabs", () => {
     saveDashboard();
     expectGoodSnowplowEvents(PAGE_VIEW_EVENT + 2); // dashboard_tab_deleted
   });
+
+  it("should send snowplow events when cards are moved between tabs", () => {
+    visitDashboard(ORDERS_DASHBOARD_ID);
+
+    expectGoodSnowplowEvents(PAGE_VIEW_EVENT);
+
+    editDashboard();
+    createNewTab();
+    goToTab("Tab 1");
+
+    expectGoodSnowplowEvents(PAGE_VIEW_EVENT);
+
+    moveDashCardToTab({ tabName: "Tab 2" });
+
+    expectGoodSnowplowEvents(PAGE_VIEW_EVENT + 1);
```

Comment:
Please another helper https://github.com/metabase/metabase/blob/92312aba6932c33f00aadd15b06763237098939b/e2e/test/scenarios/collections/uploads.cy.spec.js#L124, where the event name is specified